src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main extends FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 6;
+    private static final int FIRST_AMOUNT = 3;
+
+    public static void main(String[] agr) {
+        FigureSupplier supplier = new FigureSupplier();
+        Figure[] figures = new Figure[NUMBER_OF_FIGURES];

```

Comment:
Consider using a more descriptive variable name instead of 'supplier' to reflect the role of this object, such as 'figureSupplier'.